spec/tree-view-spec.coffee

```diff
@@ -663,7 +674,10 @@ describe "TreeView", ->
 
         children = root1.find('.directory')
         expect(children.length).toBeGreaterThan 0
-        children.each (index, child) -> expect(child).toHaveClass 'expanded'
+        children.each (index, child) ->
+          expect(child).toHaveClass 'expanded'
+          expect(child.hasAttribute('draggable')).toBe(true)
+          expect(child.getAttribute('draggable')).toBe('true')
```

Comment:
I think the `getAttribute` assertion would be sufficient. In our test framework, Jasmine, failures don't halt the test, so it will still proceed to execute the next assertion anyway.